extensions/AzureOpenAI/AzureOpenAIConfig.cs

```diff
@@ -74,6 +74,15 @@ public enum APITypes
     /// </summary>
     public int? EmbeddingDimensions { get; set; }
 
+    /// <summary>
+    /// Old AI model like ada (https://learn.microsoft.com/en-us/azure/ai-services/openai/reference) does
+    /// not support in azure version more than 16 element in a single batch, while openai accepts up to
+    /// 2k. This settings is needed to avoid sending too large batches to the service and having an error.
+    ///
+    /// Openai Has much larger windows as explained here https://platform.openai.com/docs/api-reference/embeddings/create.
```

Comment:
Could you clarify? The size of text is a different limit than the size of the batch.
OpenAI text-embedding-ada-002 accepts in input strings 8192 tokens.

Reply:
From documentation, Ada in azure accepts a batch of Max 16 elements, regardless the number of tokens